include/mbedtls/mbedtls_config.h

```diff
@@ -1414,6 +1414,23 @@
  */
 //#define MBEDTLS_PSA_CRYPTO_SPM
 
+/**
+ * \def MBEDTLS_PSA_KEY_SLOT_DYNAMIC
```

Comment:
I don't like the name. I wanted it to be close to `MBEDTLS_PSA_KEY_SLOT_COUNT`, but I don't think “key slot dynamic” or “key slot: dynamic” makes sense. How do others feel about `MBEDTLS_PSA_KEY_STORE_DYNAMIC`? Or would you prefer `MBEDTLS_PSA_DYNAMIC_KEY_STORE`?

Reply:
I'm OK with the proposed names, but I don't have a strong preference for one of the 2.